api_design/DDS-974-enforce_upload_chunk_size.md

```diff
@@ -0,0 +1,54 @@
+# DDS-974 Enforce Upload Chunk Size
+
+## Deployment View
+
+N/A
+
+## Logical View
+
+A critical issue was recently exposed with the chunked upload capability.  The underlying storage system (Swift), can only coalesce a large file that is <= 1000 segments (i.e. referred to as chunks in the DDS API).  To remedy this situation, the API will be extended to enforce a minumum chunk size for chunked file uploads.
```

Comment:
we should probably avoid mentioning 1000 specifically, as it might confuse others into thinking that this cannot be changed. Maybe say something to the effect that each storage provider is configured with a set maximum number of segments, beyond which it cannot coalesce a large file.

Reply:
Good point - I will update now and push